toolchain/check/handle_function.cpp

```diff
@@ -135,7 +135,7 @@ static auto MergeFunctionRedecl(Context& context,
 
 // Check whether this is a redeclaration, merging if needed.
 static auto TryMergeRedecl(Context& context, Parse::AnyFunctionDeclId node_id,
-                           SemIR::InstId prev_id,
+                           SemIR::InstId prev_id, SemIR::LocId prev_name_loc_id,
```

Comment:
Double checking but isn't this the new name location?
```suggestion
                           SemIR::InstId prev_id, SemIR::LocId name_loc_id,
```

Reply:
Done, thanks for catching this!